lib/OpenQA/Worker/Pool.pm

```diff
@@ -62,6 +62,7 @@ sub check_qemu_pid {
 
 sub clean_pool() {
     return if $nocleanup;
+    return unless $pooldir;
```

Comment:
Without this, an unset `$pooldir` would make `clean_pool()` attempt to remove `/`

Reply:
https://github.com/os-autoinst/openQA/blob/master/t/24-worker.t#L65 :)

Reply:
"funny" observing CI crying and being stabbed 🗡  in the attempt to remove `/`